src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.command.ByeCommand;
+import duke.command.Command;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+/**
+ * The main Duke class that represents the chatbot.
+ * It initializes the application and interacts with the user.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs a Duke object and initializes the UI, Storage, and TaskList.
+     *
+     * @param filePath The path to the file where tasks are saved and loaded.
+     * @throws FileNotFoundException If the file is not found.
+     */
+    public Duke(String filePath) throws FileNotFoundException {
+        ui = new Ui();
+        ui.showWelcome();
+        storage = new Storage(filePath);
+        tasks = new TaskList(storage.loadTasks());
+    }
```

Comment:
It seems to me that while well-handled, the bot does not start if the file has not already been created (and the only way out might be to manually create a starting file). Is this a deliberate design decision?